abstract/Block.js

```diff
@@ -134,20 +135,21 @@ export class Block extends BaseComponent {
   /** @private */
   __bindBasicCssData() {
     if (!Block._cssDataBindingsList.includes(this.ctxName)) {
-      let unprefixedCfgProps = [
-        'pubkey',
-        'store',
-        'multiple',
-        'multiple-min',
-        'multiple-max',
-        'max-files',
-        'accept',
-        'confirm-upload',
-        'init-activity',
-        'done-activity',
+      let cfgProps = [
+        '--cfg-pubkey',
+        '--cfg-store',
+        '--cfg-multiple',
+        '--cfg-multiple-min',
+        '--cfg-multiple-max',
+        '--cfg-max-files',
+        '--cfg-accept',
+        '--cfg-img-only',
+        '--cfg-confirm-upload',
+        '--cfg-init-activity',
+        '--cfg-done-activity',
```

Comment:
The only reason why I've made it prefixed - it's much easier to find specific option usages  in the whole codebase when it's prefixed

Reply:
+ agree